### ISSUE TITLE
"A Mouldy Grotto" Suggestion

### ISSUE THREAD

**Issue by Karkus476** [SuperTux/data#46]
We could make the level dark(er), and offer the player an earth powerup, so that they can see.

**Comment by maxteufel** [comment]
General idea is good. Don't make it too dark, so players without the powerup can see too (but earth powerup might make finding a secret area easier?). You should look in the Forest World Contest submissions and @serano01's "Old Forest" add-on draft for levels that could work.

**Comment by maxteufel** [comment]
Cc: @SuperTux/level-designers

**Comment by tobbi** [comment]
The problem with the earth powerup is that, as soon as the player loses it, it's gone and they can't see... This is something to keep in mind.

**Comment by maxteufel** [comment]
@tobbi 

> Don't make it too dark, so players without the powerup can see too (but earth powerup might make finding a secret area easier?).

**Comment by maxteufel** [comment]
Also, keep in mind the _stone_ feature the earth flower allows for (keep holding action and press down). This could be used as well.

**Comment by maxteufel** [comment]
I particularly like @Rusty-Box' "Deeper Down" from his Forest Island redesign submission and @serano01's "Under the forest" from the "Old Forest" addon which could be used in Forest Island or serve as idea for a redesigned "Mouldy Grotto".

**Comment by Rusty-Box** [comment]
There is a part, where you leave the cave. Making the level darker would look weird, when you leave the cave and it's still dark. My idea for solving this is: Making the level at night, or removing the cave exit and continue the cave.

**Comment by Karkus476** [comment]
@Rusty-Box Good point. We should remove the cave exit, or use this idea on a different level.
